source/python/neuropod/backends/mxnet/executor.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2020 UATC, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import os
+import json
+import mxnet as mx
+from mxnet.gluon import nn
+
+from neuropod.backends.neuropod_executor import NeuropodExecutor
+
+
+class MxnetNeuropodExecutor(NeuropodExecutor):
+    """
+    Executes a MXnet neuropod
+    """
+    def __init__(self, neuropod_path):
+        """
+        Load a MXnet neuropod
+
+        :param  neuropod_path:      The path to a MXnet neuropod package
+
+        """
+        super(MxnetNeuropodExecutor, self).__init__(neuropod_path)
+
+        # load inputs name
+        with open(os.path.join(neuropod_path, "0", "config.json"),
+                  "r") as config_file:
+            config = json.load(config_file)
+
+        neuropod_data_path = os.path.join(neuropod_path, "0", "data")
+        symbol_path = os.path.join(neuropod_data_path, "model-symbol.json")
```

Comment:
Can we hard code the file name `model-symbol`? I am not familiar with MXNet.

Reply:
We rename users `symbol_file` in `source/python/neuropod/backends/mxnet/packager.py`.